kubernetes/helm_charts/core/nginx-public-ingress/values.j2

```diff
@@ -932,7 +932,7 @@ nginxconfig: |
                         ' $request_time $upstream_response_time $pipe'
                         ' "$http_referer" "$http_user_agent" "$sb_request_id"'
                         ' "$http_x_device_id" "$http_x_channel_id" "$http_x_app_id"'
-                        ' "$http_x_app_ver" "$http_x_session_id"';
+                        ' "$http_x_app_ver" "$http_x_session_id" {{nginx_additional_log_fields}}';
```

Comment:
If I dont use this variable, will your below ruby expression work? You may need a dummy placeholder like space or make the parser below to check for field?